### ISSUE TITLE
Warnings from 1.3.0 libCBLForestDBStorage.a binary

### ISSUE THREAD

**Issue by pasin** [couchbase/couchbase-lite-ios#1374]
I have observed a lot of warning messages like below:

```
ld: warning: URGENT: building for iOS simulator, but linking in object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBLForestBridge.o)) built for tvOS. Note: This will be an error in the future.
ld: warning: object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBLForestBridge.o)) was built for newer tvOS version (9.0) than being linked (7.0)
ld: warning: URGENT: building for iOS simulator, but linking in object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBL_ForestDBDocEnumerator.o)) built for tvOS. Note: This will be an error in the future.
ld: warning: object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBL_ForestDBDocEnumerator.o)) was built for newer tvOS version (9.0) than being linked (7.0)
ld: warning: URGENT: building for iOS simulator, but linking in object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBL_ForestDBStorage.o)) built for tvOS. Note: This will be an error in the future.
ld: warning: object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBL_ForestDBStorage.o)) was built for newer tvOS version (9.0) than being linked (7.0)
ld: warning: URGENT: building for iOS simulator, but linking in object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBL_ForestDBQueryEnumerator.o)) built for tvOS. Note: This will be an error in the future.
ld: warning: object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBL_ForestDBQueryEnumerator.o)) was built for newer tvOS version (9.0) than being linked (7.0)
ld: warning: URGENT: building for iOS simulator, but linking in object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBL_ForestDBViewStorage.o)) built for tvOS. Note: This will be an error in the future.
ld: warning: object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBL_ForestDBViewStorage.o)) was built for newer tvOS version (9.0) than being linked (7.0)
ld: warning: URGENT: building for iOS simulator, but linking in object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBLForestDBStorage_vers.o)) built for tvOS. Note: This will be an error in the future.
ld: warning: object file (/Users/pasin/Git/couchbase/ToDoLite-iOS/Frameworks/libCBLForestDBStorage.a(CBLForestDBStorage_vers.o)) was built for newer tvOS version (9.0) than being linked (7.0)
```
--------
* Version:   1.3.0
* Client OS: iOS 9.1 iPhone 6 Simulator
* Server:    NA

**Comment by pasin** [comment]
The warning started from build 1.3.0-28 which includes https://github.com/couchbase/couchbase-lite-ios/commit/d9e1c6a543b1ce756da4aa856a06ef7eb105f843 commit.

**Comment by pasin** [comment]
The binary built on my local mac with XCode 7.3 works fine. Jenkins is using XCode 7.2 now.

**Comment by pasin** [comment]
From investigation with @michaelkwok yesterday, upgrading to XCode 7.3 might solve the issue and @michaelkwok is experimenting on that. Cc: @sethrosetter

**Comment by pasin** [comment]
Closed as the issue has been fixed by @michaelkwok in 1.3.0-54.
